### PR TITLE
feat: add @lavamoat/allow-scripts to lock down install scripts

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

**Comment by shanejonas** [comment]
fails on `yarn lint`

need to add `eslint` to this repo with our new configs

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+public/
```
Comment (rekmarks) [inline .eslintignore:1]: Per the module template, this should be in an `ignorePatterns` array in `.eslintrc.js`.

**Review by rekmarks — CHANGES_REQUESTED** [review]
Just a repo structure nit, otherwise looks good!

File: .eslintignore (hunk 1 above)
Comment (shanejonas) [reply .eslintignore:1]: i tried `ignorePatterns` with:

```
public/**/*.js
```

still included it, not sure the magical incantation

**Review by rekmarks — APPROVED** [review]
LGTM!

**Pull request merged**